src/include/rawrtc.h

```diff
@@ -4,7 +4,9 @@
 
 #include <stdlib.h> // TODO: Why?
 #include <stdbool.h> // bool
+#ifndef _WIN32
```

Comment:
Is this an ifdef we can safely rely on? I'd prefer a define coming from platform detection in the Cmake file.